Dev-Corner/tutorials/data_source_integration.md

```diff
@@ -3,26 +3,19 @@ Data Source Integration
 
 :grey_question: ***Questions***
 
-- *Major question that would be addressed in this tutorial (mostly general biological questions)*
-- *Second question*
-- *Third question*
-- *...*
+- *Is it possibile to import data into Galaxy from an external database?*
+- *Is available any ready-to-use example?*
+
 
 :dart: ***Objectives***
 
-- *First objective of this tutorial (It is a single sentence describing what a learner will be able to do once they have sat through the lesson. The objectives must be technical, but also theoretical, objectives. You can check [SWC lessons](http://swcarpentry.github.io/instructor-training/19-lessons/) to help you writing learning objectives.)*
-- *Second objective*
-- *Third objective*
-- *...*
+- *At the end of this lesson, the learner will have imported a dataset into Galaxy from an external data resource using his native interface*
 
 :heavy_check_mark: ***Requirements***
 
 - *Galaxy introduction*
-- *Second requirement*
-- *Third requirement*
-- *...*
 
-:hourglass: ***Time estimation*** *1d/3h/6h*
+:hourglass: ***Time estimation*** *10m*
```

Comment:
Oh I see, I'm on a wrong track. This is describing howto get data and not develop data sources isn't it? I guess this should then be as an introduction and not under Dev-Corner